src/hooks/useArbTokenBridge.ts

```diff
@@ -470,13 +459,11 @@ export const useArbTokenBridge = (
         true
       )
 
-      const tokenData = await bridge.getAndUpdateL1TokenData(
+      const tokenData = await bridge.l1Bridge.getL1TokenData(
         tokenAddress as string
       )
-      const symbol =
-        (tokenData && tokenData.ERC20 && tokenData.ERC20.symbol) || '??'
-      const decimals =
-        (tokenData && tokenData.ERC20 && tokenData.ERC20.decimals) || 18
+      const symbol = tokenData.symbol || '??'
+      const decimals = tokenData.decimals || 18
```

Comment:
```suggestion
      const symbol = tokenData.symbol
      const decimals = tokenData.decimals
```
these values are never undefined